examples/step-20/doc/intro.dox

```diff
@@ -249,7 +249,7 @@ program we do that in the following way:
     for (unsigned int i=0; i<dofs_per_cell; ++i)
       for (unsigned int j=0; j<dofs_per_cell; ++j)
         local_matrix(i,j) += (fe_values[velocities].value (i, q) *
- 		              k_inverse_values[q] *
+                              k_inverse_values[q] *
```

Comment:
Oh, how did that happen!? I thought we had purged tabs a long time ago...

Reply:
Apparently not in the doc/intro.dox and doc/results.dox files.